app/services/middleware_topology_service.rb

```diff
@@ -1,30 +1,27 @@
 class MiddlewareTopologyService < TopologyService
+  include UiServiceMixin
```

Comment:
Got it. Although it won't break anything by having it, I understand it's unnecessary. What is the suggestion to be used as the icons argument, since we are not using it, `{}` ?